prac_08/unreliable_car.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404/CP5632 Practical
+Unreliable Car class
+"""
+from prac_08.car import Car
+from random import randint
+
+
+class UnreliableCar(Car):
+    """Specialised version of a Car that includes reliability factor."""
+
+    def __init__(self, name, fuel, reliability):
+        """Initialise a UnreliableCar instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        """Drive car based on reliability comparison."""
+        reliability_comparison = randint(0,100)
+        if reliability_comparison <= self.reliability:
+            distance_driven = super().drive(distance)
+        else:
+            distance_driven = 0
```

Comment:
See https://github.com/CP1404/Practicals/blob/b7249925ab57f041e158bceca650b6b52a3fe1a6/prac_08/unreliable_car.py#L23
We should still call the super()'s drive method.